harper-core/src/linting/expr_linter.rs

```diff
@@ -11,7 +11,7 @@ use super::{Lint, Linter};
 /// boundaries.
 #[blanket(derive(Box))]
 pub trait ExprLinter: LSend {
-    /// A simple getter for the pattern to be searched for.
+    /// A simple getter for the expression you want Harper to search for.
```

Comment:
This always confuses me. Would a contributor ever use this or is it just used internally?